src/AccessibilityInsights.Actions/Actions/ListenAction.cs

```diff
@@ -42,73 +36,43 @@ public class ListenAction:IDisposable
         /// <param name="config"></param>
         /// <param name="ec"></param>
         /// <param name="listener"></param>
-        private ListenAction(RecorderSetting config, ElementContext ec, HandleUIAutomationEventMessage listener)
+        private ListenAction(TreeScope listenScope, ElementContext ec, HandleUIAutomationEventMessage listener)
```

Comment:
This TreeScope comes from UIAutomationClient - we could define our own type & map internally, but it looks like ai-win is already working with this type. Happy to hear suggestions here